Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -672,6 +670,7 @@ BoardInit (
   case ReadyToBoot:
     if ((GetBootMode() != BOOT_ON_FLASH_UPDATE) && (GetPayloadId() == 0)) {
       ProgramSecuritySetting ();
+      ClearFspHob ();
     }
```

Comment:
Better moving to EndOfFirmware callback since there is last call to FSP here.

Reply:
@gdong1 updated. fyi